run_monitor.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-
-echo "Starting monitor..."
-while true; do node ./monitor.js /repository/; done
```

Comment:
The benefit of this shell script is that it would re-start the process even in out-of-memory situations.

Reply:
Ah ok!

Reply:
Restored in 15232cd9bf408aeb759bb36239521d81f5e1c897